packages/@aws-cdk/aws-glue/lib/table.ts

```diff
@@ -185,7 +185,29 @@ export interface TableProps {
   readonly enablePartitionFiltering?: boolean;
 
   /**
-   * The user-supplied properties for the description of the physical storage of this table.
+   * The user-supplied properties for the description of the physical storage of this table. These properties help describe the format of the data that is stored within the crawled data sources.
+   *
+   * There are reserved keys that are used by AWS Glue. They CAN be mutated, but they are best left alone.
```

Comment:
Which keys are reserved?

Reply:
I've updated this again, there are a list of values that are updated by the crawler, provided by AWS. I have included the list as a link to a webpage where AWS details these.